### PR TITLE
feat: Implement custom switch button

### LINKED ISSUE #5417: Switch component
**Design:** [Figma](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/s2-Mobile?node-id=40%3A1185)
Screenshot 2022-05-17 at 19 55 13

**Spec:** 
1. Recommended props
```tsx
type SwitchProps = {
	value: boolean,
	onChange: (value: boolean) => void
	isDisabled?: boolean,
}
```
2. Consider using https://reactnative.dev/docs/switch but you will probably need to create component from scratch because of limited styling
3. Don't forget to add animations

Maybe we can use Skia to create this component?

### PR DESCRIPTION
**Pull request by juriczech**
https://tppr.me/xCXkl

Closes [#5417](https://github.com/trezor/trezor-suite/issues/5417)

### CODE REVIEW

File: packages/atoms/src/Switch.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    extend: {
+        condition: isChecked,
+        style: {
+            alignItems: 'flex-end',
+            backgroundColor: utils.colors.green,
+        },
+    },
+}));
+
+const SWITCH_CIRCLE_SIZE = 20;
```
Comment (Nodonisko) [inline packages/atoms/src/Switch.tsx:32]: empty line after pls

File: packages/atoms/src/Switch.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+export const Switch = ({ isChecked, onChange, isDisabled = false }: SwitchProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const handlePress = () => {
+        if (isDisabled) return;
+        onChange(!isChecked);
+    };
+
+    return (
+        <Pressable onPress={handlePress} style={applyStyle(wrapperStyle, { isChecked })}>
```
Comment (Nodonisko) [inline packages/atoms/src/Switch.tsx]: TouchableOpacity or Highlight please

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: packages/atoms/src/Switch.tsx (hunk 1 above)
Comment (juriczech) [reply packages/atoms/src/Switch.tsx:32]: [d2ab08d](https://github.com/trezor/trezor-suite/pull/5471/commits/d2ab08dc48c5b8e22186429b3116b2b858480ce9)

File: packages/atoms/src/Switch.tsx (hunk 2 above)
Comment (juriczech) [reply packages/atoms/src/Switch.tsx]: [d2ab08d](https://github.com/trezor/trezor-suite/pull/5471/commits/d2ab08dc48c5b8e22186429b3116b2b858480ce9)

**Review by bouchja1 — APPROVED** [review]

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**